roku-test.js

```diff
@@ -207,6 +207,49 @@ RokuTest.prototype.install = function(zip) {
   this.processQueue();
 };
 
+/**
+ * @param  {[fs.createWriteStream()]}    [destination writeStream where the image will be saved]
+ * @param  {Function} fn            [callback function]
+ */
+RokuTest.prototype.takeScreenshot = function(writeStream, fn) {
+    this.commandQueue.push(function(callback) {
+      var url = 'http://' + this.host + '/plugin_inspect';
+      var formData = {
+        mysubmit: 'Screenshot',
+        passwd: '',
+        archive: ''
+      };
+      var auth = {
+        user: 'rokudev',
+        pass: this.devPassword,
+        sendImmediately: false
+      };
+
+      request.post({url: url, formData: formData, auth: auth}, function(e, r, b) {
+          // console.log('Taking screenshot returned ' + r.statusCode);
+          this.saveScreenshot(auth, writeStream, fn);
+          callback(e);
+      }.bind(this));
+    }.bind(this));
+
+    this.processQueue();
+};
+
+RokuTest.prototype.saveScreenshot = function(auth, writeStream, fn) {
+  this.commandQueue.push(function(callback) {
+    var path = '/pkgs/dev.jpg?time=1470066056';
```

Comment:
Thanks for the patch. Roku's package utility actually generates a new time argument each time the request is sent.